src/styles/main.scss

```diff
@@ -1,3 +1,58 @@
+@import "./variables";
+
 body {
   margin: 0;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  width: $column-width;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  gap: $gap;
+
+  @for $month-length from 28 through 31 {
+    &--#{$month-length} :nth-child(#{$month-length}) {
```

Comment:
The month-length modifier will not work as expected. It will hide all days after the specified day, but it will not hide the specified day itself. You should target from nth-child(n+1) instead of nth-child(n).

Reply:
sorry, I don`t understand.... if month have 28 days, so day 28th must be visible, but not 29th